packages/babel-compat-data/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "@babel/compat-data",
+  "version": "0.0.0",
+  "author": "The Babel Team (https://babeljs.io/team)",
+  "license": "MIT",
+  "description": "",
+  "repository": "https://github.com/babel/babel/tree/master/packages/babel-compat-data",
+  "publishConfig": {
+    "access": "public"
+  },
+  "exports": {
+    "./plugins": "./data/plugins.json",
+    "./native-modules": "./data/native-modules.json",
+    "./corejs2-built-ins": "./data/corejs2-built-ins.json",
+    "./overlapping-plugins": "./data/overlapping-plugins.json"
+  },
+  "scripts": {
+    "build-data": "./scripts/download-compat-table.sh; node ./scripts/build-data.js; node ./scripts/build-modules-support.js; node ./scripts/build-overlapping-plugins.js"
+  },
+  "keywords": [
+    "babel",
+    "babel-plugin"
```

```suggestion
    "compat"
```
`babel-plugin` may be misleading here.